src/Sylius/Component/Locale/Provider/TranslationLocaleProvider.php

```diff
@@ -9,7 +9,7 @@
 * file that was distributed with this source code.
 */
 
-namespace Sylius\Bundle\CoreBundle\Locale;
+namespace Sylius\Component\Locale\Provider;
 
 use Sylius\Component\Locale\Context\LocaleContextInterface;
 use Sylius\Component\Translation\Provider\LocaleProviderInterface;
```

Comment:
Oh yeah, I see that there are two different `LocaleProviderInterface`. I've moved it to the Translation component with the intention of having it later moved into Resource. Then anyone wanting to use this class would need to install both `sylius/resource` and `sylius/locale` rather than copying it or installing the whole CoreBundle.